### DISCUSSION TITLE
Adding the Slovenian language?

### DISCUSSION THREAD

**Discussion by Vanja-S** [Dokploy/dokploy#165] [Ideas]
# Idea

Hi Dokploy team!

I've been using Dokploy for a few projects now and I love its simplicity. I thought about contributing and thought wouldn't Slovenian docs be a an idea? Slovenian is not really widely used, however its really enforced in our faculty systems even in CS bachelors or majors, where it helps to have native literature at disposal (especially if its part of course material). 

DevOps and deployment in general are really lacking in our curriculum, although I know there was a controversy whether the faculty should actually teach industry standard tools, I completely take the stance that a small practical portion should be part of this curriculum. Seeing that Dokploy is open-source and simple, I would gladly se it used to get students in touch with application deployment, Docker, etc. 

# Contribution

I'm more than happy to help and contribute to this idea! 🙌

**Comment by Siumauricio** [comment]
Hi, thank you very much for the appreciation of using dokploy for your daily use and definitely things like this did not exist when I was in college and it is not taught either and I think it is very important to encourage this as it helps many beginners to deploy their projects in a very simple way, I also went through these problems when I was in college!

Referring to the documentation in slovenian, I could accept it without any problem however to the documentation I want to do something quite big in the sense of creating many examples, explain more in detail how dokploy works internally and others and I think that if you did the documentation then it would be like lost time, what do you think if you wait to improve the documentation to be much more complete and thus to implement the slovenian language, however if you want I can accept any PR at any time, just keep in mind that detail!

**Reply by Vanja-S** [reply]
I would be happy to contribute at any time 😄, even helping with completing the documentation.
Do you have any PR open for extending the docs?

**Reply by Siumauricio** [reply]
This is the repository: https://github.com/Dokploy/docs/tree/main

you can find a example there since we have in english and chinese, so you can have an idea how to add another language

**Reply by Vanja-S** [reply]
Great, I will start working on the PR 😄
